cmd/bnscli/common.go

```diff
@@ -9,3 +14,47 @@ func sequenceID(n uint64) []byte {
 	binary.BigEndian.PutUint64(b, n)
 	return b
 }
+
+// writeTx serialize the transaction using a protocol buffer. First bytes
+// written contain the information how much space the transaction takes.
+// Size information is required to be able to stream the messages:
```

Comment:
Ahhh.... very nice. You don't use repeated after all. but best practices for streams in order to support batch

:tipping_hand_man: